server/src/main/java/com/objectcomputing/checkins/services/reports/ReportKudos.java

```diff
@@ -0,0 +1,15 @@
+package com.objectcomputing.checkins.services.reports;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.time.LocalDate;
+
+@AllArgsConstructor
+@Getter
+class ReportKudos {
+  private final LocalDate dateCreated;
+  private final String message;
+  private final String sender;
+}
+
```

Comment:
This can also be a record

```suggestion
package com.objectcomputing.checkins.services.reports;

import java.time.LocalDate;

record ReportKudos(
        LocalDate dateCreated,
        String message,
        String sender
) {
}
```